README.md

```diff
@@ -101,8 +102,23 @@ The released version of the SDK can be downloaded from https://github.com/Appdyn
     * [C++ SDK and Sample App](https://github.com/Appdynamics/iot-cpp-sdk)
     * [Python Sample Code](https://github.com/Appdynamics/iot-rest-api-sample-apps)
 
+## Miscellaneous
```

Comment:
## Code Coverage 

It's not Miscellaneous because the section is specifically for code coverage.